.github/workflows/build-ampd-release.yaml

```
@@ -170,6 +174,7 @@ jobs:
           done
 
       - name: Upload binaries to release
+        if: ${{ github.event.inputs.dry-run == 'false' }}
```

```suggestion
        if: ${{ github.event.inputs.dry-run }}
```
If it's a boolean, could you just do this instead (and take the negation)? Boolean is more strict and so preferable

---

we are treating it as string in the input. I was originally using the boolean and negation but Steven pointed out that we had failures with that in the past. We are already using this pattern here: https://github.com/axelarnetwork/axelar-amplifier/blob/main/.github/actions/release/action.yaml#L149